client/my-sites/invites/invite-form-header/index.jsx

```diff
@@ -20,6 +20,12 @@ export default React.createClass( {
 		);
 	},
 
+	getSiteName() {
+		const { site } = this.props;
+
+		return site.title || '';
+	},
```

Comment:
Where is this new function being used? 
also why not just 
`return this.props.site || this.props.site.title;`